tb_plugin/test/test_profiler.py

```diff
@@ -1043,6 +1048,11 @@ def test_dump_gpu_metrics(self):
             data_expected = file.read()
         self.assertEqual(data_with_gpu_metrics_flat, data_expected)
```

Comment:
the compare could fail if we manually change the file but the content is same?

Reply:
https://stackoverflow.com/questions/25851183/how-to-compare-two-json-objects-with-the-same-elements-in-a-different-order-equa

Reply:
Got it. Done. Thanks.